internal/controlplane/handlers_authz.go

```diff
@@ -394,7 +397,15 @@ func (s *Server) inviteUser(
 		return nil, status.Errorf(codes.Internal, "error creating invitation: %v", err)
 	}
 
-	// TODO: Publish the event for sending the invitation email
+	// Publish the event for sending the invitation email
```

Comment:
Non-blocking comment: I feel like we should move this invite logic into an `InviteService` type interface.

Reply:
thanks, this was already one of the follow-up AIs on my list 👍